src/content/docs/apm/agents/python-agent/supported-features/python-message-queues.mdx

```diff
@@ -114,15 +121,3 @@ The [**Transaction trace summary** page](/docs/apm/transactions/transaction-trac
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/) > APM > (select an app) > (select a transaction trace):** In this example, the selected transaction trace's **Summary** shows RabbitMQ operations in the colored bar chart. The **Slowest components** section also lists the RabbitMQ `Put` and `Take` results.
 </figcaption>
-
```

Comment:
Why did we remove this?

Reply:
This was an old feature in the APM that has been removed since the writing of the Pika RabbitMQ instrumentation, so I just took it out.